packages/serverpod_serialization/lib/src/serialization.dart

```diff
@@ -6,7 +6,7 @@ import 'dart:typed_data';
 import 'package:serverpod_serialization/serverpod_serialization.dart';
 
 /// The constructor takes JSON structure and turns it into a decoded
-/// [SerializableEntity].
+/// [SerializableModel].
 typedef constructor<T> = T Function(
     dynamic jsonSerialization, SerializationManager serializationManager);
```

Comment:
For later: This should probably be cleaned out as well since it might be a breaking change and I suspect it is not required anymore.